.circleci/config.yml

```diff
@@ -171,16 +171,28 @@ jobs:
           path: ios/PurchasesHybridCommon/test_output
           destination: scan-output
 
-  deploy-ios:
+  deploy-ios-phc:
     <<: *base-ios-job
     steps:
       - checkout
       - revenuecat/install-gem-mac-dependencies:
           cache-version: v1
       - trust-github-key
       - run:
-          name: Deploy new version to Cocoapods
-          command: bundle exec fastlane ios push_pods
+          name: Deploy PurchasesHybridCommon to Cocoapods
+          command: bundle exec fastlane ios push_pod_PHC
+          no_output_timeout: 30m
+
+  deploy-ios-phcui:
```

Comment:
This needs to depend on the first one

Reply:
Does this need to wait for the other pod to propagate? I think it will try to build and fail because it won't find the pod

Reply:
Yup, updated.